tests/tests_core/test_experiment.py

```diff
@@ -5,10 +5,15 @@
 
 from expan.core.experiment import Experiment
 from expan.core.results import Results
-from expan.core.util import generate_random_data
+from expan.core.util import generate_random_data, remove_model_pkls
 from tests.tests_core.test_results import mock_results_object
 
 
+# Removes .pkl compiled model files after the module tests are finished
+def tearDownModule():
```

Comment:
It would be great if you use triple quotes (docstring) for documenting method...